app/pages/Dashboard/Families/FamilyDetail/FamilyDetail.tsx

```diff
@@ -56,7 +60,7 @@ export const FamilyDetail = () => {
           { tab: <Title order={5}>اعضا</Title>, value: '2' },
         ]}
         panels={[
-          { panel: 'First Panel', value: '1' },
+          { panel: <HouseholderIdentityForm />, value: '1' },
           { panel: 'Second Panel', value: '2' },
```

Comment:
now seeing this part I think I could have done two things better

1. we can merge these two props into one, and do this separation (panel being separated from tabs) inside the component itself (to become implementation detail)
2. the `value` props could be renamed to something more meaning full like `name` perhaps

@erfan-goodarzi @ASafaeirad if you guys agree on this I can fix it

Reply:
Why not keep it simple?

Reply:
isn't that more simple?

Reply:
Why u didn't implement it like this? [tabs](https://mantine.dev/core/tabs/#usage)

Reply:
because the styling had problems
if u see in the custom Tabs I added activeTab to have control over it

Reply:
I saw it now. In that case, I agree with u.

Reply:
Let's have it for now, and have a separate session to discuss it.